src/main/resources/api.yaml

```diff
@@ -207,3 +313,15 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  UserInProjectDto:
```

Comment:
na froncie potrzebne jest coś więcej do wyświetlenia, niż samo id (dane osobowe a może nawet email?)

Reply:
To może by na to zrobić API /users, zamiast zwracać wszędzie informacje o userze zamiast userId? :thinking:

Reply:
Jest na `/users` zadanie (w sumie aktualnie na jakieś `/me`, ale przyświeca im ten sam cel, bo i tak front ma mieć id użytkownika)

Reply:
No ale jeśli dostanę liste userów w projekcie, to muszę zrobić n zapytań o dane osobowe. Zwracanie tego za jednym razem jest chyba wygodniejsze

Reply:
Więc jak proponujesz to rozwiązać?

Reply:
Pobierając skład zespołu, najwygodniej byłoby dostać (id, dane, status)

Inną opcją jest zwracanie (id, status) + ileś zapytań o dane usera, ale to kilkakrotnie wydłuży czas ładowania.

Jeszcze inną jest metoda która przyjmuje listę userId i zwraca ich dane.

Mnie najbardziej podoba się metoda pierwsza (bo jest najprostsza, najszybsza i na pewno zawsze będzie działać)